apps/aefate/src/aefa_chain_api.erl

```diff
@@ -242,6 +253,38 @@ account_balance(Pubkey, #state{primop_state = PState,
             end
     end.
 
+-spec next_nonce(pubkey(), state()) -> 'error' |
+                                       {'ok', integer(), state()}.
+next_nonce(Pubkey, #state{primop_state = PState0,
+                          onchain_primop_state = Onchain} = S) ->
+    case aeprimop_state:find_account(Pubkey, PState0) of
+        {Account, PState1} ->
+            Nonce = aec_accounts:nonce(Account) + 1,
+            case eval_primops(
+                   [aeprimop:force_inc_account_nonce_op(Pubkey, Nonce)],
+                   S#state{onchain_primop_state = {onchain, PState1}}) of
```

Comment:
You are putting `primop_state` to `onchcain_primeop_state` is this correct?

Reply:
uh, good catch